src/components/ShowAllJobs/ShowAllJobs.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import { DispatchType, RootState } from "../../redux/store/store";
+import { getAllJobs } from "../../redux/reducer/jobs";
+import { useDispatch, useSelector } from "react-redux";
+
+const ShowAllJobs = () => {
+  const { allJobs } = useSelector((state: RootState) => state.jobReducer);
+  const dispatch: DispatchType = useDispatch();
+
+ 
+  useEffect(() => {
+    const getjobFromApi = () => {
+      dispatch(getAllJobs());
+    };
+    getjobFromApi();
+  },[dispatch]);
```

Comment:
dispatch(getAllJobs());
No need to use fn.

Reply:
i have already fixed